tests/xharness/TestTasks/BuildToolTask.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.DotNet.XHarness.iOS.Shared.Execution;
+
+namespace Xharness.TestTasks {
+
+	public class BuildToolTask
+	{
+		public IProcessManager ProcessManager { get; private set; }
+		public TestPlatform Platform { get; set; }
+
+		public bool SpecifyPlatform { get; set; } = true;
+		public bool SpecifyConfiguration { get; set; } = true;
```

Comment:
It would be cool to check initializations of this class and figure out whether these fields can also be made readonly

Reply:
Ill add this as an enhancement issue, since is not the goal of the PR.